client/app/pages/dashboards/public-dashboard-page.js

```diff
@@ -18,6 +23,21 @@ const PublicDashboardPage = {
     this.logoUrl = logoUrl;
     this.public = true;
     this.dashboard.widgets = Dashboard.prepareDashboardWidgets(this.dashboard.widgets);
+
+    const refreshRate = Math.max(30, parseFloat($location.search().refresh));
+
+    if (refreshRate) {
+      const refresh = () => {
+        loadDashboard($http, $route).then((data) => {
+          this.dashboard = data;
+          this.dashboard.widgets = Dashboard.prepareDashboardWidgets(this.dashboard.widgets);
+
+          $timeout(refresh, refreshRate * 1000.0);
```

Comment:
Just interesting - why `1000.0`, not just `1000`? :-)

Reply:
No idea 😳